### PR TITLE
CUJ Tweaks for I know where to find help

### LINKED ISSUE #1157: Change numbered links to text links for /docs/webstore/faq/
Links in https://developer.chrome.com/docs/webstore/faq/ should be text links, not numeric ones. And listed at the end alphabetically.

@AmySteam will take care of this during her [CUJ CWS] project

### LINKED ISSUE #1332: [CWS CUJ] I know where to find help
[Friction Log](https://docs.google.com/document/d/14h_aqOgjUjRwH73PKST2HqWEuQY_VWUhIeOsHhKWKpI/edit#heading=h.j4f4zcsvf3wk)

### PR DESCRIPTION
**Pull request by AmySteam**
Fixes #1157 
Fixes #1332 

CC: @CharyCh 

Changes proposed in this pull request:

- Improve CWS FAQs
- Add helpful links in the intro
- Change numbered link to text links

⭐ Staged links
- [CWS FAQ](https://deploy-preview-1366--developer-chrome-com.netlify.app/docs/webstore/faq/)
- [What's the CWS](https://deploy-preview-1366--developer-chrome-com.netlify.app/docs/webstore/about_webstore/)

### CODE REVIEW

File: site/en/docs/webstore/faq/index.md [hunk 1]
```diff
@@ -129,36 +125,41 @@ You may also report fake reviews via the Developer [Support Form][cws-support].
...
-Visit [https://chrome.google.com/webstore/devconsole][110] and click on the listing you wish to
+To [Update your Chrome Web Store item][update-item], visit the [developer dashboard][dev-dashboard] and click on the listing you wish to
 update. Once you upload a new version and click **Publish**, your update will automatically be
 pushed out to users over the next few hours.
 
 ### Can I host my own extension? {: #faq-dev-03 }
 
-Yes, you may host your own extension, although you will need to take care of [packaging and
-serving][111] the [auto-update manifest][112] yourself.
+Yes, you can host your own extension. However, Windows and Mac OS
```
Comment (samthor) [inline site/en/docs/webstore/faq/index.md:161]: Mac OS => macOS

File: site/en/docs/webstore/faq/index.md [hunk 2]
```diff
@@ -129,36 +125,41 @@ You may also report fake reviews via the Developer [Support Form][cws-support].
...
+To [Update your Chrome Web Store item][update-item], visit the [developer dashboard][dev-dashboard] and click on the listing you wish to
 update. Once you upload a new version and click **Publish**, your update will automatically be
 pushed out to users over the next few hours.
 
 ### Can I host my own extension? {: #faq-dev-03 }
 
-Yes, you may host your own extension, although you will need to take care of [packaging and
-serving][111] the [auto-update manifest][112] yourself.
+Yes, you can host your own extension. However, Windows and Mac OS 
+Users can only install extensions hosted in the Chrome Web store, except for installs via [enterprise publishing][cws-enterprise] or developer mode.
```
Comment (samthor) [inline site/en/docs/webstore/faq/index.md:162]: Users => users

File: site/en/docs/webstore/faq/index.md (hunk 1 above)
Comment (samthor) [inline site/en/docs/webstore/faq/index.md:161]: This "yes" is kind of bad considering the vast majoity of users are excepted. Can you rewrite this?

e.g., "You can host your own extensions in limited cases, such as for Linux or Chrome OS users. Consider [enterprise publishing] or developer mode."

File: site/en/docs/webstore/faq/index.md [hunk 3]
```diff
@@ -174,7 +175,7 @@ The item won't appear in the web store until it has successfully completed the r
 details, see [How long will it take to review my item?][how-long-do-reviews-take].
 
 Extensions which are published to the same domain as the publisher address may be approved more
-quickly.
+quickly. Learn more about [Enterprise publishing][enterprise].
```
Comment (samthor) [inline site/en/docs/webstore/faq/index.md:178]: Above you capitalize Enterprise, here you don't. I suspect lowercase is better.

File: site/en/docs/webstore/about_webstore/index.md [hunk 4]
```diff
@@ -21,11 +21,13 @@ build Chrome-specific features into it.
...
 
-[1]: http://chrome.google.com/webstore
-[3]: https://chrome.google.com/webstore/developer/dashboard
-[4]: /docs/extensions/mv3/overview/
-[5]: /docs/webstore/publish
+[cws-faq]: /docs/webstore/faq/
+[cws-publish]: /docs/webstore/publish
+[dashboard]: https://chrome.google.com/webstore/developer/dashboard
+[extensions]: /docs/extensions/mv3/overview/
+[webstore]: http://chrome.google.com/webstore
```
Comment (samthor) [inline site/en/docs/webstore/about_webstore/index.md:32]: update to https:// (we should probably automate this, but you're here changing stuff)

**Review by samthor — APPROVED** [review]

File: site/en/docs/webstore/faq/index.md (hunk 3 above)
Comment (AmySteam) [reply site/en/docs/webstore/faq/index.md:178]: Good catch!

File: site/en/docs/webstore/faq/index.md (hunk 1 above)
Comment (AmySteam) [reply site/en/docs/webstore/faq/index.md:161]: Sounds much better! I'll change it :)

**Pull request merged**